ttnn/cpp/ttnn/operations/eltwise/binary_ng/device/binary_ng_device_operation.cpp

```diff
@@ -76,6 +76,21 @@ void BinaryNgDeviceOperation::validate_on_program_cache_miss(
             "Second operand to eltwise binary must be interleaved");
     }
 
+    if (!tensor_args.output_tensor.has_value()) {  // Inplace operation
+        const auto& input_tensor_a = tensor_args.input_tensor_a;
+        const auto& output_shape = compute_output_specs(attributes, tensor_args).shape();
```

Comment:
is there another function called compute_broadcasted shape ?

Reply:
No. Broadcasted shape is being computed inside compute_output_specs.